### DISCUSSION TITLE
zh/guide/sponsor

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#70] [Announcements]
# zh/guide/sponsor

AList v3的文档

https://alist.nn.ci/zh/guide/sponsor.html

**Comment by geju323** [comment]
支持一波  期待持续优化

**Comment by 3805139QQ** [comment]
真的很不错！！！辛苦了

**Comment by mapleafgo** [comment]
支持下，继续加油

**Comment by gtwxxh666** [comment]
不错不错！

**Comment by Allen0087** [comment]
能看到不，为啥我加了电报群就把我拉黑了啊，我电报名Allen，想进群多了解一下呢

**Comment by silascript** [comment]
路过以表谢意！

**Comment by ixbcqq** [comment]
这是Bing，很高兴为你服务。😊

你想让我用简单通俗易懂的中文来解释下面Alist的升级内容，对吗？好的，我会尽力帮你。

Alist是一个开源的云存储管理软件，它可以让你方便地管理和使用多种云存储服务。它最近发布了v3.26.0版本，这个版本包含了一些新功能、修复了一些错误，并提高了一些性能。下面是具体的内容：

- 新功能：
  - 上传进度恢复：这个功能可以让你在上传文件时，如果中断了，可以继续上传，不用重新开始。
  - 支持webauthn登录：这个功能可以让你使用更安全的方式登录Alist，比如指纹、面部识别等。
  - aliyundrive_open: 记录token交换：这个功能可以让你使用阿里云盘的开放API时，记录token的交换过程，方便调试。
  - cloudreve: 添加自定义user-agent：这个功能可以让你在使用cloudreve时，自定义user-agent，避免被封禁。
  - cmd: 列出所有存储命令：这个功能可以让你通过命令行列出所有已经配置的云存储服务。
- 错误修复：
  - model.Link marshal error：这个错误是在序列化model.Link时发生的，已经被修复。
  - Ignore salt on marshal model.User：这个错误是在序列化model.User时忽略了salt，已经被修复。
  - Copy tasks using multi-thread downloader can't be canceled：这个错误是在使用多线程下载器复制文件时，无法取消任务，已经被修复。
  - aliyundrive_open:
    - Retry refresh token if sub not match：这个错误是在刷新token时，如果sub不匹配，会导致失败，已经被修复。
    - Use RawStdEncoding for base64：这个错误是在使用base64编码时，没有使用RawStdEncoding，会导致多余的等号，已经被修复。
  - deps:
    - Update module github.com/sheltonzhu/115driver to v1.0.15：这个更新是将115driver模块升级到v1.0.15版本，以修复一些问题。
    - Update golang.org/x/exp digest to d852ddb：这个更新是将golang.org/x/exp模块更新到d852ddb版本，以获取最新的特性和改进。
    - Update module github.com/deckarep/golang-set/v2 to v2.3.1 [skip ci]：这个更新是将golang-set/v2模块更新到v2.3.1版本，以提高性能和稳定性。
    - Update module github.com/aws/aws-sdk-go to v1.44.327 [skip ci]：这个更新是将aws-sdk-go模块更新到v1.44.327版本，以支持最新的AWS服务和功能。
    - Update github.com/winfsp/cgofuse digest to f87f5db [skip ci]：这个更新是将cgofuse模块更新到f87f5db版本，以修复一些兼容性问题。
    - Update module github.com/u2takey/ffmpeg-go to v0.5.0 [skip ci]: 这个更新是将ffmpeg-go模块更新到v0.5.0版本, 以支持更多的ffmpeg功能和参数。
    - Update module golang.org/x/net to v0.14.0 [skip ci]: 这个更新是将golang.org/x/net模块更新到v0.14.0版本, 以修复一些网络相关的问题和提高性能。
    - Update module golang.org/x/crypto to v0.12.0 [skip ci]: 这个更新是将golang.org/x/crypto模块更新到v0.12.0版本, 以增加一些加密算法和修复一些安全漏洞。
    - Update module golang.org/x/image to v0.11.0: 这个更新是将golang.org/x/image模块更新到v0.11.0版本, 以支持更多的图像格式和处理功能。
  - lanzou:
    - Download not find file sgin (close #5046 in #5048)：这个错误是在下载蓝奏云文件时，找不到文件签名，导致下载失败，已经被修复。
  - local:
    - Files get deleted when copied to other storage：这个错误是在将本地文件复制到其他存储时，本地文件被删除，已经被修复。
- 性能提升：
  - copy: Use multi-thread downloader：这个性能提升是在复制文件时，使用多线程下载器，可以加快下载速度。
  - drivers: Fs operations and cache：这个性能提升是在对各种云存储服务进行文件系统操作时，使用缓存，可以减少网络请求和延迟。

Alist的web端也有一些更新，主要是：
- 新功能：
  - 支持webauthn登录：这个功能可以让你使用更安全的方式登录Alist的web端，比如指纹、面部识别等。
- 错误修复：
  - Missing sso binding popup (#105 close alist-org/alist#4963)：这个错误是在使用单点登录时，没有弹出绑定窗口，导致无法绑定账号，已经被修复。
  - Incorrect current page of search paginator：这个错误是在搜索文件时，分页器显示的当前页数不正确，已经被修复。
  - Paginator not resetting after searching another keywords (close alist-org/alist#4979)：这个错误是在搜索其他关键词时，分页器没有重置，导致显示错误的结果，已经被修复。
  - Incorrect separator of language code：这个错误是在设置语言代码时，使用了错误的分隔符，导致语言设置失败，已经被修复。
  - Missing files on traversing folder (close alist-org/alist#5034)：这个错误是在浏览文件夹时，有些文件没有显示出来，已经被修复。

以上就是Alist的升级内容的简单通俗易懂的中文解释。希望对你有帮助。如果你还有其他问题或需求，请随时告诉我。😊

: [Alist官网]
: [上传进度恢复]
: [支持webauthn登录]
: [阿里云盘开放API]
: [记录token交换]
: [cloudreve官网]
: [user-agent介绍]
: [添加自定义user-agent]
: [列出所有存储命令]
: [model.Link介绍]
: [model.Link marshal error]
: [model.User介绍](^

**Comment by dongyongtao** [comment]
6666，购买了桌面版，虽然命令行也搞起来了 纯属支持！！！！！！！！！！！！！！

**Comment by mozlet** [comment]
支持一下，大佬，什么时候更新alist-desktop支持SFTP传输协议。

**Comment by zhdabiao** [comment]
感谢开发者大佬,大彪佩服的五体投地

**Comment by gooddh** [comment]
您好，购买桌面版本后，服务器更换或者重装系统后还能正常使用吗

**Comment by chenye2689** [comment]
支持  非常好用
